ledger/stateproofverificationtracker.go

```diff
@@ -183,7 +204,13 @@ func (spt *stateProofVerificationTracker) LookupVerificationData(stateProofLastA
 		errStateProofVerificationDataNotFound)
 }
 
+// isn't thread safe. should be called with stateProofVerificationMu held.
```

Comment:
I think it is fair to state that this is not a thread-safe function, but if we add a comment, let's also give some hints for what this function does.